Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +181,16 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTION_OPTIONS[
+                                 self.loss_function]
```

Comment:
What about
```py
items['Loss function'] = \
    self.LOSS_FUNCTION_OPTIONS[self.loss_function]
```
If you rename `LOSS_FUNCTION_OPTIONS` to `LOSS_FUNCTIONS` (and `LEARNING_RATES` and `PENALTIES` likewise), the it all even fits on a single line.